pkg/importer/dataStream.go

```diff
@@ -224,13 +205,9 @@ func CopyImage(dest, endpoint, accessKey, secKey string) error {
 		return errors.Wrapf(err, "unable to create data stream")
 	}
 	defer ds.Close()
-	return ds.copy(dest)
+	return ds.Copy(dest)
```

Comment:
Why is this exported? `CopyImage` is the main exposed api. Are there external users of this method?

Reply:
That is a left over from an experiment where I needed access to the data stream in order to flush some stuff, I don't think I need it anymore